src/store/state.js

```diff
@@ -50,6 +51,21 @@ export default {
     filters: [],
     options: []
   },
+  covid19network: {
+    filters: [],
+    options: [
+      { id: covid19BiobankNetworkSelectionId, label: 'Biobanks providing COVID-19 services' },
+      { id: covid19CollectionNetworkSelectionId, label: 'Collections with COVID-19 samples' }
+    ]
+  },
+  // this is a shadow facet, used to filter out biobanks on network id for covid19.
+  biobank_network: {
+    filters: []
+  },
+  // this is a shadow facet, used to filter out collections on network id for covid19.
```

Comment:
It's following the normal facet flow, so that it is inline with the other facets and the rsql builder. To not make things more complicated

Reply:
Test improvement was necessary as the state changes would mean fixing it in more than 3 places. and making it worse in the other tests. But otherwise agreed.